### PR TITLE
replace flavourMap by flavourDB

### PR DESCRIPTION
**Pull request by tornado404**

### CODE REVIEW

File: pkg/job/runtime/kubernetes/executor/vcjob.go [hunk 1]
```diff
@@ -37,6 +37,7 @@ const (
 
 type VCJob struct {
 	KubeJob
+	Flavour schema.Flavour
```
Comment (D0m021ng) [inline pkg/job/runtime/kubernetes/executor/vcjob.go]: It might be better to get `Flavour` from tasks for single pod.

File: cmd/server/main.go [hunk 2]
```diff
@@ -124,15 +123,6 @@ func initConfig() error {
 		return err
 	}
 
-	ServerConf.FlavourMap = make(map[string]schema.Flavour)
```
Comment (D0m021ng) [inline cmd/server/main.go:127]: Add init sql to `installer/database/paddleflow.sql`

**Review by D0m021ng — APPROVED** [review]

File: pkg/job/runtime/kubernetes/executor/vcjob.go (hunk 1 above)
Comment (tornado404) [reply pkg/job/runtime/kubernetes/executor/vcjob.go]: removed

File: cmd/server/main.go (hunk 2 above)
Comment (tornado404) [reply cmd/server/main.go:127]: done

**Pull request merged**